site/index.xml

```diff
@@ -187,10 +187,10 @@ limitations under the License.
               <h2>Commercial Distribution</h2>
               <p>
                 VMware offers a <a href="https://tanzu.vmware.com/rabbitmq">range of commercial offerings for RabbitMQ</a>.
-                This includes a distribution called <a href="https://tanzu.vmware.com/services-marketplace/messaging-and-integration/rabbitmq">Tanzu RabbitMQ</a>,
-                a version that deploys in <a href="https://tanzu.vmware.com/tanzu">VMware Tanzu platform</a>,
-                and a forthcoming <a href="https://tanzu.vmware.com/content/blog/introducing-rabbitmq-for-kubernetes">version for Kubernetes</a>.
-                These distributions include all of the features of the open source version, with some additional management features. Support agreements are part of the commercial licensing.
+                This includes a distribution called <a href="https://tanzu.vmware.com/services-marketplace/messaging-and-integration/rabbitmq">Tanzu RabbitMQ</a> that deploys on Kubernetes or
```

Comment:
I think this information needs further clarity how about:

As well as open source RabbitMQ, VMware offers a range of commercial offerings for RabbitMQ which you can search for at [Tanzu RabbitMQ](https://tanzu.vmware.com). These offerings include all of the features of the open source version, with some additional management and business continuity features. Support agreements are part of the commercial licensing.

One of these offerings is [Tanzu RabbitMQ for Kubernetes](https://network.tanzu.vmware.com/products/p-rabbitmq-for-kubernetes) that deploys on Kubernetes or your container Runtime. Another offering is a version that deploys in [VMware Tanzu Application Service](https://network.tanzu.vmware.com/products/elastic-runtime).

Reply:
If I was a customer I would ask "why are they telling me about these offerings instead of others" what is so great about these? Maybe we should add a 2 liner of what these offerings can do for them here or else link the blog other sources of information where they can find that out